lib/beaker/options/presets.rb

```diff
@@ -121,6 +121,11 @@ def presets
           :project                => 'Beaker',
           :department             => 'unknown',
           :created_by             => ENV['USER'] || ENV['USERNAME'] || 'unknown',
+          :host_tags              => {
```

Comment:
I believe that you are going to have to do some work here to get the env var support for these values to still work correctly, otherwise they will get stored as :department instead of host_tags[:department].